pep-0676.rst

```diff
@@ -121,69 +128,84 @@ The following redirect rules must be created from the `python.org`_ domain:
-any old URL scheme will be guaranteed to work.
+Due to server-side redirects to new canonical URLs, links in previously
+published materials referring to the old URL scheme will be guaranteed to work.
+Furthermore, aside from the resolved bugs and new implemented features, all
+existing PEPs will still render as they did before with the new backend.
+Therefore, this poses no major backwards compatibility concerns.
+
 
 Security Implications
 =====================
 
-No security implications, and the main `python.org`_ website will no longer
-take raw HTML uploads, closing a potential threat vector.
+The main `python.org`_ website will no longer need to process raw HTML
+uploads, closing a potential threat vector, and the PEP building and
+deployment process will use modern, well maintained code and secure
+automated platforms, reducing the likely potential attack surface. Therefore,
+there is no forseen negative security impact.
+
 
 How to Teach This
 =================
 
 The new canonical URLs will be publicised in the documentation. However, this
 is mainly a backend infrastructure change, and there should be minimal
-end-user impact.
+end-user impact. PEP 1 and PEP 12 will be updated as needed to include and
+document any relevant new roles, directive, syntax and other constructs enabled
+by this change.
+
 
 Reference Implementation
 ========================
 
 The proposed implementation has been merged into the `python/peps`_ repository
-in a series of pull requests [8]_. This automatically renders all PEPs on every
-commit.
+in a series of pull requests [8]_. It currently automatically renders all
+PEPs on every commit.
+
 
 Rejected Ideas
 ==============
 
-It would likely be possible to amend the current (as of November 2021)
-rendering process to include a lot of the quality-of-life improvements and
-issue mitigations mentioned above. However, we do not believe that this would
+It would likely be possible to amend the current docutils-based manual
+rendering process to include many of the quality-of-life improvements and
+bug mitigations mentioned above. However, we do not believe that this would
 solve the distributed tooling issue.
 
 It would be possible to use the output from the proposed rendering system and
-import it into `python.org`_. We would argue however that this would be the
-worst of both worlds, as a great deal of complexity is added, and none is
-removed.
+import it into the existing `python.org`_ domain. We would argue, however,
+that this would be the worst of both worlds, as a great deal of complexity
+is added, while none is removed.
+
 
 Open Issues
 ===========
 
 None.
 
-Acknowledgements
-================
 
-Thanks to Hugo van Kemenade, Pablo Galindo Salgado, and Éric Araujo for support
-since April 2020.
+Acknowledgments
+===============
+
+Thanks to Hugo van Kemenade, Pablo Galindo Salgado and Éric Araujo for
+supporting this effort since April 2020.
```

Comment:
In its previous usage, "for support" is an unambiguous grammar error. At the very minimum it needs a definite article, like this:

```suggestion
Thanks to Hugo van Kemenade, Pablo Galindo Salgado, and Éric Araujo for the
support since April 2020.
```

and I tweaked it further to avoid the awkward and mildly ambiguous phrasing, but its not _wrong_ without that, per say. If you'd prefer a more conservative but still correct change, the above would suffice.